bugsnag-plugin-android-ndk/src/test/java/com/bugsnag/android/NativeEventDecoder32bitTest.kt

```diff
@@ -95,4 +99,46 @@ class NativeEventDecoder32bitTest {
     private fun verifyUserInfoDecode() {
         verify(event, times(1)).setUser("999999", "ndk override", "j@ex.co")
     }
+
+    private fun verifyErrorDecode() {
+        val stackFrame1 = Stackframe(null, null, null, null)
+        val stackFrame2 = Stackframe(null, null, null, null)
+
+        val error = event.errors.single()
+        assertEquals("SIGSEGV", error.errorClass)
+        assertEquals("Segmentation violation (invalid memory reference)", error.errorMessage)
+        assertEquals(ErrorType.UNKNOWN, error.type)
+
+        stackFrame1.frameAddress = 1285807130L
+        stackFrame1.symbolAddress = 1285807124L
+        stackFrame1.loadAddress = 1285804032L
+        stackFrame1.lineNumber = 3098L
+        stackFrame1.file = "/data/data/com.example.bugsnag.android/lib/libentrypoint.so"
+        stackFrame1.method = "Java_com_example_bugsnag_android_BaseCrashyActivity_crashFromCXX"
+        stackFrame1.codeIdentifier = "5ddb429dfa12daf935fbe29b6d2d498a5740e0eb"
+
+        stackFrame2.frameAddress = 1082654128L
+        stackFrame2.symbolAddress = 1082654016L
+        stackFrame2.loadAddress = 1082527744L
+        stackFrame2.lineNumber = 126384L
+        stackFrame2.file = "/system/lib/libdvm.so"
+        stackFrame2.method = "dvmPlatformInvoke"
+        stackFrame2.codeIdentifier = ""
```

Comment:
I don't think we need either of these `Stackframe` objects the expected values can rather move into the `assert` calls